jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/CassandraIntegrationTest.java

```diff
@@ -44,110 +41,74 @@ protected JmxScraperContainer customizeScraperContainer(
-                "1"),
-        metric ->
-            assertSum(
-                metric,
-                "cassandra.storage.total_hints.in_progress.count",
-                "Number of hints attempting to be sent currently",
-                "1",
-                /* isMonotonic= */ false),
-        metric ->
-            assertSumWithAttributes(
-                metric,
-                "cassandra.client.request.count",
-                "Number of requests by operation",
-                "1",
-                attrs -> attrs.containsOnly(entry("operation", "RangeSlice")),
-                attrs -> attrs.containsOnly(entry("operation", "Read")),
-                attrs -> attrs.containsOnly(entry("operation", "Write"))),
-        metric ->
-            assertSumWithAttributes(
-                metric,
-                "cassandra.client.request.error.count",
-                "Number of request errors by operation",
-                "1",
-                getRequestErrorCountAttributes()));
+  protected MetricsVerifier createMetricsVerifier() {
+    return MetricsVerifier.create()
+        .assertGauge(
+            "cassandra.client.request.range_slice.latency.50p",
+            "Token range read request latency - 50th percentile",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.range_slice.latency.99p",
+            "Token range read request latency - 99th percentile",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.range_slice.latency.max",
+            "Maximum token range read request latency",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.read.latency.50p",
+            "Standard read request latency - 50th percentile",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.read.latency.99p",
+            "Standard read request latency - 99th percentile",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.read.latency.max",
+            "Maximum standard read request latency",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.write.latency.50p",
+            "Regular write request latency - 50th percentile",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.write.latency.99p",
+            "Regular write request latency - 99th percentile",
+            "us")
+        .assertGauge(
+            "cassandra.client.request.write.latency.max",
+            "Maximum regular write request latency",
+            "us")
+        .assertCounter(
+            "cassandra.compaction.tasks.completed",
+            "Number of completed compactions since server [re]start",
+            "1")
+        .assertGauge(
+            "cassandra.compaction.tasks.pending",
+            "Estimated number of compactions remaining to perform",
+            "1")
+        .assertUpDownCounter(
```

Comment:
[question to revier] In original framework we had assertSum method with overloaded version accepting isMonotonic parameter added. This was little unclear to me for two reasons:
1) It was not obvious that assertSum without isMonotonic parameter was the same as assertSum(isMonotonic=true)
2) In YAML we used instrument types gauge, counter, updowncounter, while in assertions we used assertGauge, assertSum with and without isMonotonic parameter. 

That's why I decided to implement assertGauge, assertCounter and assertUpDownCounter method families here.
I know it may lead to more methods to be implemented but for me this new API is cleaner.

What do you think about this approach?